docs/src/tutorials/pino_ode.md

```diff
@@ -0,0 +1,95 @@
+# Physics Informed Neural Operator for ODEs
+
+This tutorial provides an example of how to use the Physics Informed Neural Operator (PINO) for solving a family of parametric ordinary differential equations (ODEs).
+
+## Operator Learning for a family of parametric ODEs
+
+In this section, we will define a parametric ODE and solve it using a PINO. The PINO will be trained to learn the mapping from the parameters of the ODE to its solution.
+
+```@example pino
+using Test # hide
+using OptimizationOptimisers
+using Lux
+using Statistics, Random
+using LuxNeuralOperators
+using NeuralPDE
+
+equation = (u, p, t) -> p[1] * cos(p[2] * t) + p[3]
+tspan = (0.0f0, 1.0f0)
+u0 = 1.0f0
+prob = ODEProblem(equation, u0, tspan)
+
+number_of_parameter = 3
+deeponet = LuxNeuralOperators.DeepONet(
+    Chain(
+        Dense(number_of_parameter => 10, Lux.tanh_fast), Dense(10 => 10, Lux.tanh_fast), Dense(10 => 10)),
+    Chain(Dense(1 => 10, Lux.tanh_fast), Dense(10 => 10, Lux.tanh_fast),
+        Dense(10 => 10, Lux.tanh_fast)))
+
+u = rand(3, 50)
+v = rand(1, 40, 1)
+θ, st = Lux.setup(Random.default_rng(), deeponet)
+c = deeponet((u, v), θ, st)[1]
+
+bounds = [(1.0f0, pi), (1.0f0, 2.0f0), (2.0f0, 3.0f0)]
+number_of_parameter_samples = 50
+strategy = StochasticTraining(40)
+opt = OptimizationOptimisers.Adam(0.03)
+alg = PINOODE(deeponet, opt, bounds, number_of_parameters; strategy = strategy)
```

Comment:
I think this needs to explained step by step as its not intuitive to a user. So breakdown step by step with some small explanations.